Update labeling across the documentation (#402)
zkSNACKs/WasabiDoc: docs/why-wasabi/10Commandments.md

Each time you receive coins, you will be asked to create a label.
This label is only for you and is stored exclusively on your device.

> Wasabi has strong coin control features, and as you continue to use the wallet, you will observe that the history of your coins will appear, and this history is only useful to you if you are practicing good labeling.

Reviewer: What is displayed is not the `history` of your coins but the how known are your coins. This is, how many people knows about each of them or how private they are.

Reviewer 2: @RiccardoMasutti this

> Wasabi has strong coin control features, and as you continue to use the wallet, you will observe that the history of your coins will appear, and this history is only useful to you if you are practicing good labeling.

Reviewer 2: ```suggestion
Wasabi has strong coin control features, and as you continue to use the wallet, you can see how known are your coins, which is possible only if you practice good labeling.
```
